File: OOP_variable/4.py
class C(object):
    def __init__(self, v):
        #__를 변수 앞에 붙이면 메소드 밖에서 접근불가
        self.__value = v
    def show(self):
        print (self.__value)
c1=C(10)
# print(c1.value)
# c1.__value = 20
c1.show()

File: OOP_variable/4.rb
class C
  #메소드 밖에서 인스턴스에 직접 접근 가능하게
  # attr_reader :value
  # attr_writer :value
  attr_accessor :value
  def initialize v
    @value = v
  end
  def show
    p @value
  end
end
c1 = C.new 10
p c1.value
c1.value = 20
p c1.value
